src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1694,11 +1687,12 @@ private void AggregateRenderPackages(IEnumerable<HelixRenderPackage> packages)
                         }
 
                         AddLabelPlace(baseId, p.Positions[0], rp);
-                        pointGeometry3D.Geometry = points;
+                        if (pointGeometry3D.Geometry == null)
+                        {
+                            pointGeometry3D.Geometry = points;
+                        }
                         pointGeometry3D.Name = baseId;
```

Comment:
Do we need to put this line into the if for consistency?

Reply:
no, the name should always be updated as `baseId` has been set correctly in all cases as far as I can tell - the two cases are:
1. this is a new GeometryModel 
2. this is an existing GeometryModel that we have pushed more data to.

Reply:
Gotcha